File: src/androidTest/java/poetry/android/test/java/data/models/Group.java
package poetry.android.test.java.data.models;

import com.j256.ormlite.field.DatabaseField;
import com.j256.ormlite.table.DatabaseTable;

import poetry.annotations.MapFrom;

@DatabaseTable
public class Group
{
    @DatabaseField(id = true, columnName = "id")
	@MapFrom("id")
    public int id;

    @DatabaseField(columnName = "name")
	@MapFrom("name")
    private String name;

	public int getId()
	{
		return id;
	}

	public String getName()
	{
		return name;
	}
}

File: src/androidTest/java/poetry/android/test/kotlin/data/models/Group.kt
package poetry.android.test.kotlin.data.models

import com.j256.ormlite.field.DatabaseField
import com.j256.ormlite.table.DatabaseTable
import poetry.json.annotations.MapFrom

@DatabaseTable
class Group(
	@DatabaseField(id = true, columnName = "id")
	@MapFrom("id")
	val id: Int,

	@DatabaseField(columnName = "name")
	@MapFrom("name")
	val name: String
) {
	internal constructor(): this(0, "")
}
